zcash_client_backend/src/data_api/wallet.rs

```diff
@@ -407,12 +409,17 @@ where
         .get_target_and_anchor_heights(min_confirmations)
         .and_then(|x| x.ok_or_else(|| Error::ScanRequired.into()))?;
 
-    // derive the corresponding t-address
+    // derive the corresponding transparent internal ovk
+    let t_fvk: transparent::FullViewingKey = {
+        let secp = secp256k1::Secp256k1::new();
+        let pk = secp256k1::PublicKey::from_secret_key(&secp, sk);
+        (extfvk.chain_code(), pk).into()
```

Comment:
Actually, as it turns out I'm making changes to #341 that will make it so that the chain code is available, so I will resolve this issue once that's done.

Reply:
Add a TODO and/or use a chain code that is obviously wrong, like all-zeros, for now.